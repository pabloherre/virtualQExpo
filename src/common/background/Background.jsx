import React from 'react';
import background from '../../../assets/images/background.png';
import { ImageBackground } from 'react-native';
import { setTestInfo } from '../../utils/test.utils';

export default function withBackground(WrappedComponent) {
  return class WithBackground extends React.Component {
    render() {
      return (
        <ImageBackground {...setTestInfo('testImageBackground')} source={background} style={{ flex: 1, alignSelf: 'stretch' }}>
          <WrappedComponent {...this.props} />
        </ImageBackground>
      );
    }
  };
}
