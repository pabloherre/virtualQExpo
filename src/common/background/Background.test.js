import React from 'react';
import renderer from 'react-test-renderer';
import Background from './Background';

describe('<Background />', () => {
  it('renders correctly', async () => {
    const button = renderer.create(<Background />).toJSON();
    expect(button).toMatchSnapshot();
  });
});
