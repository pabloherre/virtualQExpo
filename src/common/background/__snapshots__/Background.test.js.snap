// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Background /> renders correctly 1`] = `null`;
